src/Kafka/KafkaInstances/KafkaInstances.stories.tsx

```diff
@@ -67,6 +68,7 @@ export const OrgAdminCanDeleteAnyInstance = Template.bind({});
 OrgAdminCanDeleteAnyInstance.args = {
   canChangeOwner: () => true,
   canDelete: () => true,
+  instanceSuspended: () => true,
```

Comment:
```suggestion
  canOpenConnection: () => true,
```
the idea is to remove any knowledge of the _why_ something is disabled in this component. `istanceSsspended` isn't any different from `row.status === 'suspended'`